make.js

```diff
@@ -637,47 +645,78 @@ function cleanupCSSSource(file) {
 // modern HTML5 browsers. Requires Google Closure Compiler.
 //
 target.minified = function() {
-  var compilerPath = process.env['CLOSURE_COMPILER'];
-  if (!compilerPath) {
-    echo('### Closure Compiler is not set. Specify CLOSURE_COMPILER variable');
+  try {
+    var compilerPackagePath = require.resolve('google-closure-compiler');
+    var compilerDirPath = path.dirname(compilerPackagePath);
+    var compilerPath = path.join(compilerDirPath, 'compiler.jar');
+  } catch (e) {
+    console.log('google-closure-compiler is not installed. Run "npm install" ' +
```

Comment:
Not really, I'm just used to console.log - happy to change to echo for consistency.